_observing-your-data/ad/index.md

```diff
@@ -110,7 +110,7 @@ To get the entity model size of a detector, use the [profile detector API]({{sit
 This formula provides a good starting point, but make sure to test with a representative workload.
 {: .note }
 
-For example, for a cluster with three data nodes, each with 8 GB of JVM heap size, a maximum memory percentage of 10% (default), and the entity model size of the detector as 1MB: the total number of unique entities supported is (8.096 * 10^9 * 0.1 / 1 MB ) * 3 = 2429.
```

Comment:
(8096 * 0.1 / 1 MB ) * 3 does not equal to 2429. We multiply by 10^9 since 1 MB is 10^9 bytes. Can you change to

```
(8096 MB * 0.1 / 1 MB ) * 3 = 2429
```

?

Reply:
Hi, the format you mentioned is exactly what I want to change.